src/Microsoft.IdentityModel.Tokens.Saml/SamlTokenUtilities.cs

```diff
@@ -102,5 +100,62 @@ internal static IEnumerable<SecurityKey> GetKeysForTokenSignatureValidation(stri
             }
             return null;
         }
+
```

Comment:
I know SamlTokenUtilities was added in a separate PR, but it should be internal.
I added 1022496 to our backlog to add checking for new public apis to our API compat analysis.